client/state/sites/selectors/get-customizer-url.js

```diff
@@ -12,17 +12,19 @@ import isJetpackSite from './is-jetpack-site';
  *
  * @param   {object}  state     Global state tree
  * @param   {?number} siteId    Site ID
- * @param   {string}  panel     Optional panel to autofocus
- * @param   {string}  returnUrl Optional return url for when the user closes the customizer
- * @returns {string}            Customizer URL
+ * @param   {?string} panel     Optional panel to autofocus
+ * @param   {?string} returnUrl Optional return url for when the user closes the customizer
+ * @param   {?string} guide     Optional parameter to show a help guide in the customizer
+ * @returns {string}             Customizer URL
  */
-export default function getCustomizerUrl( state, siteId, panel, returnUrl ) {
+export default function getCustomizerUrl( state, siteId, panel, returnUrl, guide ) {
```

Comment:
IMHO, I think using `guide` is a little bit strange. As I imaging when I use this function, I will assume that every customizer URL supports the guide. Using `queryArgs` as parameter or append the guide after we get the URL might be better.

Reply:
Wondering if the choice  of 'guide' was due to its pre-existing use in the backend: D64107-code

Reply:
yea it's not clear what possible values are for `guide` here. Every customizer url does technically support `guide`, it can either be `?guide` to show the generic guide or `?guide=add-menu` or `?guide=social-media`.

I think that the pattern of passing in the optional query parameter as an argument does make sense because it's what we use for `panel` and `returnUrl` which are also optional.

I'll add better docs for this function!

Reply:
@arthur791004 I've pushed an update now just to update the docs for that method

Reply:
Great 👍